From f0a83fcc3ca0080844384c1cd3692ce7f7521a1d Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sun, 15 Mar 2020 01:05:39 +0200
Subject: [PATCH 104/108] Revert: Add 2 attributes for list_divider_material

	modified:   core/res/res/drawable/list_divider_material.xml
	modified:   core/res/res/values-night/themes_device_defaults.xml
	modified:   core/res/res/values/attrs.xml
	modified:   core/res/res/values/colors_device_defaults.xml
	modified:   core/res/res/values/colors_material.xml
	modified:   core/res/res/values/themes_device_defaults.xml
	modified:   core/res/res/values/themes_material.xml

Change-Id: Ibdd2b0d4f45b2d85be42011a9f49e3585867d083
---
 core/res/res/drawable/list_divider_material.xml      | 4 ++--
 core/res/res/values-night/themes_device_defaults.xml | 3 +--
 core/res/res/values/attrs.xml                        | 4 ----
 core/res/res/values/colors_device_defaults.xml       | 2 --
 core/res/res/values/colors_material.xml              | 3 ---
 core/res/res/values/themes_device_defaults.xml       | 3 +--
 core/res/res/values/themes_material.xml              | 4 ----
 7 files changed, 4 insertions(+), 19 deletions(-)

diff --git a/core/res/res/drawable/list_divider_material.xml b/core/res/res/drawable/list_divider_material.xml
index e4c12fb1..658a59d5 100644
--- a/core/res/res/drawable/list_divider_material.xml
+++ b/core/res/res/drawable/list_divider_material.xml
@@ -15,8 +15,8 @@
 -->
 
 <shape xmlns:android="http://schemas.android.com/apk/res/android"
-       android:tint="?attr/colorListDivider">
-    <solid android:color="?attr/opacityListDivider" />
+       android:tint="?attr/colorForeground">
+    <solid android:color="#1f000000" />
     <size
         android:height="1dp"
         android:width="1dp" />
diff --git a/core/res/res/values-night/themes_device_defaults.xml b/core/res/res/values-night/themes_device_defaults.xml
index 93975a9d..75ef109f 100644
--- a/core/res/res/values-night/themes_device_defaults.xml
+++ b/core/res/res/values-night/themes_device_defaults.xml
@@ -60,8 +60,7 @@ easier.
         <item name="colorBackground">@color/primary_dark_device_default_settings</item>
         <item name="colorEdgeEffect">@color/edge_effect_device_default_dark</item>
 
-        <item name="colorListDivider">@color/list_divider_color_dark</item>
-        <item name="opacityListDivider">@color/list_divider_opacity_device_default_dark</item>
+        <item name="listDivider">@color/list_divider_color_dark</item>
     </style>
 
     <style name="Theme.DeviceDefault.Settings.DialogBase" parent="Theme.Material.BaseDialog">
diff --git a/core/res/res/values/attrs.xml b/core/res/res/values/attrs.xml
index 2f301946..ded0b6cb 100644
--- a/core/res/res/values/attrs.xml
+++ b/core/res/res/values/attrs.xml
@@ -89,10 +89,6 @@
         <attr name="backgroundDimEnabled" format="boolean" />
         <!-- Color of background imagery used for popup windows. -->
         <attr name="colorPopupBackground" format="color" />
-        <!-- Color used for list divider. -->
-        <attr name="colorListDivider" format="color" />
-        <!-- Opacity used for list divider. -->
-        <attr name="opacityListDivider" format="color" />
 
         <!-- =========== -->
         <!-- Text styles -->
diff --git a/core/res/res/values/colors_device_defaults.xml b/core/res/res/values/colors_device_defaults.xml
index 80a1b942..22c78f29 100644
--- a/core/res/res/values/colors_device_defaults.xml
+++ b/core/res/res/values/colors_device_defaults.xml
@@ -48,8 +48,6 @@
 
     <color name="list_divider_color_light">#00ffffff</color>
     <color name="list_divider_color_dark">#00ffffff</color>
-    <color name="list_divider_opacity_device_default_light">@android:color/white</color>
-    <color name="list_divider_opacity_device_default_dark">@android:color/white</color>
 
     <color name="loading_gradient_background_color_dark">#44484C</color>
     <color name="loading_gradient_background_color_light">#F8F9FA</color>
diff --git a/core/res/res/values/colors_material.xml b/core/res/res/values/colors_material.xml
index 7829e3b0..90cdee8a 100644
--- a/core/res/res/values/colors_material.xml
+++ b/core/res/res/values/colors_material.xml
@@ -149,7 +149,4 @@
     <color name="autofill_background_material_dark">@color/material_blue_grey_900</color>
     <color name="autofill_background_material_light">@color/material_grey_50</color>
 
-    <!-- List divider opacity -->
-    <color name="list_divider_opacity_material">#1f000000</color>
-
 </resources>
diff --git a/core/res/res/values/themes_device_defaults.xml b/core/res/res/values/themes_device_defaults.xml
index 2b37a0c1..fc558b1f 100644
--- a/core/res/res/values/themes_device_defaults.xml
+++ b/core/res/res/values/themes_device_defaults.xml
@@ -1488,8 +1488,7 @@ easier.
         <!-- Progress bar attributes -->
         <item name="progressBarCornerRadius">@dimen/config_progressBarCornerRadius</item>
 
-        <item name="colorListDivider">@color/list_divider_color_light</item>
-        <item name="opacityListDivider">@color/list_divider_opacity_device_default_light</item>
+        <item name="listDivider">@color/list_divider_color_light</item>
     </style>
 
     <!-- @hide DeviceDefault theme for a window that should use Settings theme colors
diff --git a/core/res/res/values/themes_material.xml b/core/res/res/values/themes_material.xml
index ce293895..6337db19 100644
--- a/core/res/res/values/themes_material.xml
+++ b/core/res/res/values/themes_material.xml
@@ -53,8 +53,6 @@ please see themes_device_defaults.xml.
         <item name="backgroundDimAmount">0.6</item>
         <item name="colorError">@color/error_color_material_dark</item>
         <item name="colorPopupBackground">?attr/colorBackground</item>
-        <item name="colorListDivider">?attr/colorForeground</item>
-        <item name="opacityListDivider">@color/list_divider_opacity_material</item>
 
         <!-- Text styles -->
         <item name="textAppearance">@style/TextAppearance.Material</item>
@@ -426,8 +424,6 @@ please see themes_device_defaults.xml.
         <item name="backgroundDimAmount">0.6</item>
         <item name="colorError">@color/error_color_material_light</item>
         <item name="colorPopupBackground">?attr/colorBackground</item>
-        <item name="colorListDivider">?attr/colorForeground</item>
-        <item name="opacityListDivider">@color/list_divider_opacity_material</item>
 
         <!-- Text styles -->
         <item name="textAppearance">@style/TextAppearance.Material</item>
-- 
2.17.1

